Gemfile.lock

```diff
@@ -63,9 +78,16 @@ PLATFORMS
   ruby
 
 DEPENDENCIES
+  color_pound_spec_reporter
+  curb
   jekyll (~> 3.8)
   jekyll-feed (~> 0.12.1)
   jekyll-paginate (~> 1.1)
   jekyll-seo-tag (~> 2.6)
   jekyll-sitemap (~> 1.3)
+  minitest
+  minitest-reporters
   rake
+
+BUNDLED WITH
```

Comment:
Checking out this branch and running `bundle install` gives me an error:

```
Traceback (most recent call last):
	4: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
	3: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
	2: from /home/john/.rvm/gems/ruby-2.5.1/bin/bundle:23:in `<main>'
	1: from /home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
```

Removing the `BUNDLED WITH` lines and running `bundle install` works.

Should we just remove these lines? I'm not familiar enough with ruby/bundler to understand what these things are doing.

Reply:
Could you run `bundle update` and see if that works?

Reply:
You may also need to run `gem install curb`. Will update the PR description and maybe add some docs to the readme or contributing page.

Reply:
Added this info to README documentation about running the tests.

Reply:
Tried

```
 bundle update
Traceback (most recent call last):
	4: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
	3: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
	2: from /home/john/.rvm/gems/ruby-2.5.1/bin/bundle:23:in `<main>'
	1: from /home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
```

and after running `gem install curb`:

```
[john] /home/john/Code/docs/r2
→ gem install curb
Building native extensions. This could take a while...
Successfully installed curb-0.9.10
Parsing documentation for curb-0.9.10
Done installing documentation for curb after 0 seconds
1 gem installed
[john] /home/john/Code/docs/r2
→ bundle install
Traceback (most recent call last):
	4: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
	3: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
	2: from /home/john/.rvm/gems/ruby-2.5.1/bin/bundle:23:in `<main>'
	1: from /home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
[john] /home/john/Code/docs/r2
→ bundle update
Traceback (most recent call last):
	4: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `<main>'
	3: from /home/john/.rvm/gems/ruby-2.5.1/bin/ruby_executable_hooks:24:in `eval'
	2: from /home/john/.rvm/gems/ruby-2.5.1/bin/bundle:23:in `<main>'
	1: from /home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:308:in `activate_bin_path'
/home/john/.rvm/rubies/ruby-2.5.1/lib/ruby/site_ruby/2.5.0/rubygems.rb:289:in `find_spec_for_exe': can't find gem bundler (>= 0.a) with executable bundle (Gem::GemNotFoundException)
```

Why do we need to add this `BUNDLED WITH` line?

Reply:
Bundler adds that line automatically (I didn't add it). What do you get when running `bundle -v`?

Reply:
I don't think it's an issue to remove it, but it's a symptom of an underlying issue.

Reply:
Here's where the `BUNDLE WITH` line came from: it's autogenerated when the file was created by bundler version 2.x instead of 1.x. Explanations and how to upgrade (tldr: run `bundle update --bundler` to upgrade): https://bundler.io/guides/bundler_2_upgrade.html